csp/gitprojectsettings.csp

```diff
@@ -55,6 +55,15 @@ hr {
  set $namespace = namespace
 
  set webuiURL = "/isc/studio/usertemplates/gitsourcecontrol/webuidriver.csp/"_$namespace_"/?CSPSHARE=1"
+ set appBasePath = "/isc/studio/usertemplates/gitsourcecontrol/"
```

Comment:
Since this code is duplicated, I think it would make sense to refactor to a method somewhere.

Reply:
Agreed. I tried it initially but I couldn't get it to work. This time, I haven't tried it. I think I have an idea though.